third_party/java/java_tools/BUILD

```diff
@@ -30,3 +19,122 @@ genrule(
     cmd = "zip -q -j $@ $$(echo $(SRCS) | sort)",
     visibility = ["//third_party:__pkg__"],
 )
+
+filegroup(
+    name = "java_tools_jars",
+    srcs = [
+        "ExperimentalRunner_deploy.jar",
+        "GenClass_deploy.jar",
+        "JacocoCoverage_jarjar_deploy.jar",
+        "JavaBuilder_deploy.jar",
+        "Runner_deploy.jar",
+        "VanillaJavaBuilder_deploy.jar",
+        "jarjar_command_deploy.jar",
+        "turbine_deploy.jar",
+        "turbine_direct_binary_deploy.jar",
+    ] + select({
+        "//src/conditions:arm": ["bazel-singlejar_deploy.jar"],
+        "//conditions:default": [],
+    }),
+    visibility = ["//visibility:private"],
+)
+
+genrule(
```

Comment:
What are these genrules for?

Reply:
You're right, I don't need them. I removed them